content/en/synthetics/api_tests/http_tests.md

```diff
@@ -55,21 +55,16 @@ After choosing to create an `HTTP` test, define your test's request.
 
    {{% tab "Authentication" %}}
 
+   Upload a private key and client certificate, select an authentication method, and provide additional configuration details as prompted.
```

Comment:
Can we keep the wording for client certificates while  moving it at the top please
https://a.cl.ly/8Lu1e6LB